README.md

```diff
@@ -106,7 +106,7 @@ There are several **enhancer** packages, which are intended to be used in conjun
   module.exports = compose(
     baseConfig('node'),
     withReactNative(),
-    withNTL(),
+    withNTL(), // ⚠️ Always after .withReactNative
```

Comment:
Maybe add the same warnings like enzyme?

Reply:
Enzyme does not introduce duplicated paths in the config, but sure!

Reply:
Done ✅